packages/web/docs/src/app/_meta.ts

```diff
@@ -108,20 +82,17 @@ const meta: Record<string, DeepPartial<Item | MenuItem | PageItem>> = {
       'about-us': {
         title: 'About Us',
         href: 'https://the-guild.dev/about-us',
-        newWindow: true,
       },
       'brand-assets': {
         title: 'Brand Assets',
         href: 'https://the-guild.dev/logos',
-        newWindow: true,
       },
     },
   },
   'graphql-foundation': {
     title: 'GraphQL Foundation',
     type: 'page',
     href: 'https://graphql.org/community/foundation/',
-    newWindow: true,
```

Comment:
Is it smart enough on it's own to detect if a link should be opened in a new window?

Reply:
this option is removed in nextra, all external links now are opened in new window